SUMMARY.md

```diff
@@ -2,24 +2,32 @@
 
 * [Tool Installations](README.md)
 
-## BROWSER
+## Tips
```

Comment:
I'd recommend moving this to the end. If we want to spell out how to split screen to make it easier, we should add the instructions just for that part to the Chrome section. The instruction for opening Git Bash isn't relevant until they install Git for example so having the instruction here may be confusing. I'll also use this page to list keyboard shortcuts.